charts/handbrake/1.0.0/test_values.yaml

```diff
@@ -0,0 +1,127 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+
+#All values here are set as the docker defaults.
+env:
+# Permissions Settings
+  USER_ID: "1000"
+  GROUP_ID: "1000"
```

Comment:
same thing here:
envTpl: 
  USER_ID: "{{ .Values.PUID }}"
  GROUP_ID: "{{ .Values.PGID }}"

Reply:
Note: Not 100% this would work right out of the box.
It "should work"(tm)

Reply:
These are (like I said) envTpl settings, NOT env settings.
I asked to move those to envTpl.